.github/workflows/releases.yml

```diff
@@ -0,0 +1,40 @@
+name: Releases
+
+on: push
+
+jobs:
+  releases:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v22
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+
+    - name: build pwndbg
+      run: nix build '.?submodules=1#pwndbg' -o result-pwndbg
+
+    - name: build rpm
+      run: nix build '.?submodules=1#rpm' -o dist-rpm
+    - name: build deb
+      run: nix build '.?submodules=1#deb' -o dist-deb
+    - name: build apk
+      run: nix build '.?submodules=1#apk' -o dist-apk
+    - name: build archlinux
+      run: nix build '.?submodules=1#archlinux' -o dist-archlinux
+    - name: build tarball
+      run: nix build '.?submodules=1#tarball' -o dist-tarball
+
+    - name: release
+      uses: softprops/action-gh-release@v1
```

Comment:
The `softprops` is a random GitHub account and so the 'v1' tag can be modified in time. 
To prevent against potential supply chain attacks where someone modifies the v1 version to publish malicious files we should pin to a git commit instead of 'v1' here.

When we do it, please point out the tag the commit comes from so it can be verified manually in the future.